makeLUTs/create_genes_dictionary.py

```diff
@@ -22,30 +22,31 @@ def build_ensembl_genes():
     '''
 
     #connect to Ensembl MySQL public server
-    core = create_engine('mysql+mysqldb://anonymous@ensembldb.ensembl.org:3337/homo_sapiens_core_93_37')
+    core = create_engine('mysql+mysqldb://anonymous@ensembldb.ensembl.org:3337/homo_sapiens_core_93_37', connect_args={'compress': True})
```

Comment:
isn't the database version hardcoded here? @Docproc ?

Reply:
It is, however when the database we're using changes, the SQL as it was will most likely not return any results, and it won't be obvious why.

Reply:
@Docproc but it needs to be configurable as we use grc38 for the main platform but grc37 for the genetics platform